purchase_request/views/purchase_request_view.xml

```diff
@@ -249,6 +252,7 @@
                     <field name="analytic_account_id"
                            groups="analytic.group_analytic_accounting"/>
                     <field name="supplier_id"/>
+                    <field name="cancelled"/>
```

Comment:
@lreficent Can we make the field "cancelled" invisible in all tree view? The line is already grey when it has been cancelled so I guess we can hide it.